src/views/ResetPassword.vue

```diff
@@ -0,0 +1,123 @@
+      style="width: 800px; height: 550px; margin-top: 13%; margin-left: 25%;"
+    >
+      <div class="card-header pb-0 text-center">
+        <h2 class="h1 mb-3">Creat new password</h2>
+        <p class="mb-5 lead">
+          Create a new password that is at least 8 characters long. A strong password has a combination of letters, digits and punctuation marks
+        </p>
+      </div>
+      <div class="card-body pt-2 px-0 px-lg-7">
+        <div class="row justify-content-center">
+          <div class="col-12 col-md-8">
+            <div class="form-group">
+              <label for="exampleInputPassword345">Password</label>
+              <div class="input-group mb-4">
+                <div class="input-group-prepend">
+                  <span class="input-group-text"
+                    ><span class="fas fa-unlock-alt"></span
+                  ></span>
+                </div>
+                <input
+                  class="form-control"
+                  id="exampleInputPassword345"
+                  placeholder="Password"
+                  type="password"
+                  v-model="password"
+                  aria-label="Password"
+                  required=""
+                />
+              </div>
+            </div>
+            <div class="form-group">
+              <label for="exampleConfirmPassword712">Confirm Password</label>
+              <div class="input-group">
+                <div class="input-group-prepend">
+                  <span class="input-group-text"
+                    ><span class="fas fa-unlock-alt"></span
+                  ></span>
+                </div>
+                <input
+                  class="form-control"
+                  id="exampleConfirmPassword712"
+                  placeholder="Confirm password"
+                  type="password"
+                  v-model="password2"
+                  aria-label="Password"
+                  required=""
+                />
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <button class="btn btn-primary" id="conbtn" type="button">Continue</button>
+    </div>
+  </div>
+</template>
+<script>
+import { config } from "@/configurations";
+
+export default {
+  name: "ResetPassword",
+  components: {},
+  data() {
+    return {
+      password: "",
+      password2: "",
+      submitted: false,
+    };
+  },
+  methods: {
+    handleSubmit() {
+      this.submitted = true;
+      const username = this.$route.params.username;
+      const { password, password2 } = this;
+      const { dispatch } = this.$store;
+      if (password && password2) {
+        if (password.trim() !== password2.trim()) {
+          dispatch(
+            "alertStore/error",
+            config.messagingConfig.messages.error.password_mismatch,
+            { root: true }
+          );
+        }
+      }
+      dispatch("authStore/resetPassword", password, username);
+  }
+  }
+};
+</script>
+
+<style>
+.yo {
```

Comment:
What does `yo` mean ? Please use proper class name ? This is not school level project.